awx/main/tasks.py

```diff
@@ -868,6 +868,21 @@ def build_params_process_isolation(self, instance, private_data_dir, cwd):
                 process_isolation_params['process_isolation_ro_paths'].append(instance.ansible_virtualenv_path)
         return process_isolation_params
 
+    def build_params_resource_profiling(self):
+        '''
+        Build ansible runner .run() parameters for resource profiling.
+        '''
+        resource_profiling_params = dict()
+        if isinstance(self.instance, Job) and getattr(settings, 'AWX_PERF_STATS_ENABLED', False):
+            logger.info('Resource profiling enabled')
+            resource_profiling_params['resource_profiling'] = True
+            # TODO: Params for setting polling interval
```

Comment:
I don't care about this as much in this PR; let's pick sane defaults that don't destroy AWX performance (.25).